packages/i18n-calypso/package.json

```diff
@@ -26,10 +26,12 @@
 		"interpolate-components": "^1.1.1",
 		"lodash": "^4.17.11",
 		"lru": "^3.1.0",
-		"react": "^16.8.3",
 		"tannin": "^1.1.1",
 		"use-subscription": "^1.2.0"
 	},
+	"peerDependencies": {
+		"react": "^16.8"
+	},
```

Comment:
This won't matter because `interpolate-components` depends directly, so we'll get `react` transitively there:

https://github.com/Automattic/interpolate-components/blob/master/package.json

Reply:
https://github.com/Automattic/interpolate-components/issues/15